packages/mobile/src/send/selectors.ts

```diff
@@ -7,3 +10,11 @@ export const getRecentPayments = (state: RootState) => {
 export const isSendingSelector = (state: RootState) => {
   return state.send.isSending
 }
+
+export const canSendTokensSelector = createSelector(
+  tokensWithTokenBalanceSelector,
+  localCurrencyToUsdSelector,
+  (tokensList, usdExchangeRate) => {
+    return tokensList.length > 0 && usdExchangeRate !== null
```

Comment:
just for my understanding, are we only checking for usdExchangeRate here (as opposed to the exchange rate for  euro or some other currency the user might have as default) because all the exchange rates are set at the same time so if one is not null then all of them will be safely not null?

Reply:
Nope, we are only using the usd exchange rate in the new flow. The EUR and CELO exchange rates are there for legacy reasons.

Right now, to get the value for a user in local currency we transform from the token to USD and then from USD to local currency.